app/javascript/packs/utils/calendar.js

```diff
@@ -6,18 +6,18 @@ const Day = Immutable.Record({day: undefined, inner: undefined, today: false});
 const Week = Immutable.Record({days: Immutable.List()});
 
 export function prev(base){
-  return base.clone().subtract(1, 'M').date(base.date());
+  return base.clone().subtract(1, 'M');
 }
 
 export function next(base){
-  return base.clone().add(1, 'M').date(base.date());
+  return base.clone().add(1, 'M');
 }
 
 export function week(date, range){
   return Immutable.Range(0, daysPerWeek).map((i)=> {
     let day = date.clone().add(i, 'd');
     let [from, to] = range;
-    let inner = day.isBetween(from, to, 'day');
+    let inner = day.isBetween(from, to, 'day', '[]');
```

Comment:
This `'[]` feels weird (with no meaning) I had to search in the Moment docs to understand what it means.
![image](https://user-images.githubusercontent.com/54550926/186964077-aebc4984-776f-4637-84bd-37f570866879.png)

What if we can let the meaning of that more explicit? I mean, something like:
```js
const inclusivity = '[]';
// mabe
// const includeBothValues = '[]';
let inner = day.isBetween(from, to, 'day', inclusivity);
```
Would that make sense? Or am I missing something?